src/test/resources/packs/simple_objects.yaml

```diff
@@ -0,0 +1,55 @@
+java:
+  Application.java: |
+    package com.example;
+    public class Application {
+      public static void main(String[] args) {
+        new B(new A(42)).bar();
+      }
+    }
+  A.java: |
+    package com.example;
+    public class A {
+        private final int x;
+        public A(int x) {
+            this.x = x;
+        }
+        public int foo() {
+            return x * 2;
+        }
+    }
+  B.java: |
+    package com.example;
+    public class B {
+      private final A a;
+      public B(A a) {
+        this.a = a;
+      }
+      public int bar() {
+        return a.foo() + 1;
+      }
+    }
+eo:
+  Application.eo: |
+    package com.example
+    [] > Application
+      [args...] > main
+        seq > @
+          (b.new (a.new 42)).bar
+  A.eo: |
+    package com.example
+    [] > A
+      cage x   
```

Comment:
@volodya-lombrozo here it should be `cage > x`

Reply:
@maxonfjvipon Done